plugins/gs.yaml

```diff
@@ -14,13 +14,13 @@ spec:
       matchLabels:
         os: darwin
         arch: amd64
-    uri: https://github.com/giantswarm/kubectl-gs/releases/download/v0.5.2/kubectl-gs-v0.5.2-darwin-amd64.tar.gz
-    sha256: 3a2a043cc4d854a9c78263b0404a2806de8a6d6e0ae5bd7c3b754df472b84624
-    bin: ./kubectl-gs-v0.5.2-darwin-amd64/kubectl-gs
+    uri: https://github.com/giantswarm/kubectl-gs/releases/download/v0.5.3/kubectl-gs-v0.5.3-darwin-amd64.tar.gz
+    sha256: efd8341e9f371a4b164f22c30584b83dc123be740e41a2f878526738b401afca
+    bin: ./kubectl-gs-v0.5.3-darwin-amd64/kubectl-gs
```

Comment:
Can you change the `files` spec into a wildcard copy to a stable location? Then our automation will approve manifests immediately.

The problem currently is that the bin field changes between versions.

Reply:
Honestly not sure why the bot didn't trigger. According to webhook logs, it didn't recognize it as a bump PR. But when I try the `go run ./bumpctl -file <(curl -sSL https://github.com/kubernetes-sigs/krew-index/pull/701.diff)` it shows the problem just fine. Let's see if the problem persists.

Reply:
We should be able to change to a wildcard. Thanks!